openapi_tester/schema_tester.py

```diff
@@ -292,15 +297,17 @@ def _test_openapi_type_object(
                     response=data,
                     schema=schema_section,
                     reference=reference,
-                    hint="You need to add the missing schema key to the response, or remove it from the documented response.",
+                    hint="You need to add the missing schema key to the response, "
+                    "or remove it from the documented response.",
                 )
             if response_key not in properties:
                 raise DocumentationError(
                     message=f"Key `{response_key}` not found in the OpenAPI schema.",
                     response=data,
                     schema=schema_section,
                     reference=reference,
-                    hint="You need to add the missing schema key to your documented response, or stop returning it in your API.",
+                    hint="You need to add the missing schema key to your documented "
```

Comment:
same principle. Concise - if we cant fit it in one line, it should be shorter :)